### DISCUSSION TITLE
node + database

### DISCUSSION THREAD

**Discussion by jdion84** [beyondcode/herd-community#161] [General]
the only thing really keeping me on valet is node and database setup

i need to use homebrew to install node & mysql anyways, so why not just use brew + valet for everything?

i prefer installing mysql on the machine rather than using dbgin so it is always running

if herd came with node + database installer options i'd make the switch today

**Comment by mpociot** [comment]
Nobody forces you to completely uninstall brew. You can still use it for the database, node and more things. Herd will Stil simplify the PHP management and updating for you 🙂

**Reply by jdion84** [reply]
Thanks for the reply

Perhaps in future versions herd can be used to manage db versions and start / stop the services as well
